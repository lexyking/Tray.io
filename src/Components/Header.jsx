import React, { Component } from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import AppBar from 'material-ui/AppBar';

export class Header extends Component {
  render() {
    return (
      <MuiThemeProvider>
        <AppBar position="static" title="Tray.io Registration form" />

      </MuiThemeProvider>
    )
  }
}

export default Header
