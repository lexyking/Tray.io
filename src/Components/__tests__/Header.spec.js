import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../Header';

// Because the header is an invariable component in this applicationCache, we can test the Snapshot
// and compare it to the previous one

describe('Header component', () => {
  test('Matches the snapshot', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  })
})