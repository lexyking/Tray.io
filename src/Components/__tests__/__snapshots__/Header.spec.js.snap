// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Matches the snapshot 1`] = `
<header
  className="header"
>
  <section
    className="logo"
  >
    <img
      alt=""
      src="tray-io.png"
    />
  </section>
  <h1>
    Joan Yassi
  </h1>
</header>
`;
